bentoml/db.py

```diff
@@ -33,9 +34,18 @@
 
 def init_db(db_url):
     # Use default config if not provided
-    engine = create_engine(
-        db_url, echo=False, connect_args={'check_same_thread': False}
-    )
+    # we have to parse the db url. Depends on what type of it,
+    from sqlalchemy_utils import database_exists
+
+    extra_db_args = {'echo': True}
+
+    if is_sqlite_db(db_url):
+        extra_db_args['connect_args'] = {'check_same_thread': False}
+        extra_db_args['echo'] = False
+    engine = create_engine(db_url, **extra_db_args)
+
+    if not database_exists(engine.url):
+        raise BentoMLException(f'Database name is missing in config db.url: {db_url}')
```

Comment:
should this message be  "Database does not exist or Database name is missing in config db.url: {db_url}"?